src/Collector/ClassNameCollector.php

```diff
@@ -9,22 +9,11 @@
 
 class ClassNameCollector implements CollectorInterface
 {
-
-
     public function getType()
     {
         return 'className';
     }
 
-    private function getRegexByConfiguration(array $configuration)
```

Comment:
i prefer to keep releated code together, i would prefer not mixing changes and code moves in one pr, it's hard to review :)